pkg/grafana/client.go

```diff
@@ -51,7 +51,9 @@ func NewClient(baseURL *url.URL, cfg gapi.Config) (*Client, error) {
 func (c *Client) LookupUser(loginOrEmail string) (gapi.User, error) {
 	user, err := c.client.UserByEmail(loginOrEmail)
 	if err != nil {
-		if !strings.Contains(err.Error(), "User not found") {
+		// the error message is longer but given that this is constrained
+		// to a returned message from an specific call, it's better to keep it short
+		if !strings.Contains(strings.ToLower(err.Error()), "user not found") {
```

Comment:
While the gapi client doesn't wrap the typed errors from grafana/pkg/models, both [8.2](https://github.com/grafana/grafana/blob/d7f71e9eaec57995d4537f758e300b31650d23ee/pkg/models/user.go#L10) and [6.5](https://github.com/grafana/grafana/blob/75f04c731f170918cfb4031ac059a7c7463f5b2e/pkg/models/user.go#L10) appear to export ErrUserNotFound errors that could be used for the string match here without depending on downcase and an expected string.

Reply:
I will adjust to using the typed errors.